tests/pragmas/tcustom_pragma.nim

```diff
@@ -345,4 +345,25 @@ type
     provider*: proc(obj: string): pointer {.dependency("Data/" & obj, 16.1), noSideEffect.}
 
 proc myproc(obj: string): string {.dependency("Data/" & obj, 16.1).} =
-  result = obj
\ No newline at end of file
+  result = obj
+
+# bug 12523
+template myCustomPragma {.pragma.}
```

Comment:
I changed the "customPragma" pragma name because it conflicts with the imported "custompragma" module symbol, not sure if that's a bug or not